src/setupTests.ts

```diff
@@ -0,0 +1,15 @@
+import { expect, afterEach } from 'vitest';
+import { cleanup } from '@testing-library/react';
+import matchers from '@testing-library/jest-dom/matchers';
+import { server } from 'mocks/server';
```

Comment:
you should use relative path to import something
```suggestion
import { server } from './mocks/server';
```